tokio/src/task/coop/mod.rs

```diff
@@ -250,14 +250,27 @@ cfg_coop! {
     use pin_project_lite::pin_project;
     use std::cell::Cell;
     use std::future::Future;
+    use std::marker::PhantomData;
     use std::pin::Pin;
     use std::task::{ready, Context, Poll};
 
+    /// Value returned by the [`poll_proceed`](poll_proceed) method.
```

Comment:
The link location isn't necessary in this case. There are quite a few of these cases that can be fixed.
```suggestion
    /// Value returned by the [`poll_proceed`] method.
```

Reply:
Learned that in the meantime in DataFusion as well which has a lint configured for this. Duplication removed.